src/CMakeLists.txt

```diff
@@ -706,6 +706,12 @@ add_subdirectory( core )
 add_subdirectory( launcher )
 add_subdirectory( dosbox-cdrom )
 
+if (WIN32) 
+        set( WIN32_SPECIFIC_LIBS dbghelp)
```

Comment:
That won't work, as it's part of the Windows SDK, which naturally isn't installed on common PCs.

Reply:
That used to be the case, but all currently supported versions of windows include dbghelp.dll (see https://docs.microsoft.com/en-us/windows/win32/debug/dbghelp-versions). 
To make sure I tried this on a PC without visual studio or the windows SDK installed and it worked.

Reply:
Ah, didn't know that, I only remember having issues with it a few years ago.